packages/experiments/src/components/Button/Button.view.tsx

```diff
@@ -29,29 +29,33 @@ export const ButtonView: IButtonComponent['view'] = props => {
     }
   };
 
-  return (
+  const Button = (keytipAttributes: any): JSX.Element => (
+    <Slots.root
+      type="button" // stack doesn't take in native button props
+      role="button"
+      onClick={_onClick}
+      {...buttonProps}
+      {...keytipAttributes}
+      disabled={disabled && !allowDisabledFocus}
+      aria-disabled={disabled}
+      tabIndex={!disabled || allowDisabledFocus ? 0 : undefined}
+      aria-label={ariaLabel}
+      ref={buttonRef}
+    >
+      <Slots.stack horizontal as="span" tokens={{ childrenGap: 8 }} verticalAlign="center" horizontalAlign="center" verticalFill>
+        <Slots.icon />
+        <Slots.content />
+        {children}
+      </Slots.stack>
+    </Slots.root>
+  );
+
+  return keytipProps ? (
     <KeytipData keytipProps={keytipProps} disabled={disabled && !allowDisabledFocus}>
-      {(keytipAttributes: any): JSX.Element => (
-        <Slots.root
-          type="button" // stack doesn't take in native button props
-          role="button"
-          onClick={_onClick}
-          {...buttonProps}
-          {...keytipAttributes}
-          disabled={disabled && !allowDisabledFocus}
-          aria-disabled={disabled}
-          tabIndex={!disabled || allowDisabledFocus ? 0 : undefined}
-          aria-label={ariaLabel}
-          ref={buttonRef}
-        >
-          <Slots.stack horizontal as="span" tokens={{ childrenGap: 8 }} verticalAlign="center" horizontalAlign="center" verticalFill>
-            <Slots.icon />
-            <Slots.content />
-            {children}
-          </Slots.stack>
-        </Slots.root>
-      )}
+      {(keytipAttributes: any): JSX.Element => <Button keytipAttributes={keytipAttributes} />}
     </KeytipData>
+  ) : (
+    <Button />
```

Comment:
I'm pretty sure using JSX here will create another `Button` element in the React hierarchy. I can't remember if we touched on this, but is there a way to make this a function call like OUFR's Button?